packages/tokens/src/tokens/color/component-dark.json

```diff
@@ -1,7 +1,7 @@
 {
-  "button-primary-background-on-dark": {
-    "name": "button-primary-background-on-dark",
-    "value": "{semantic-test-on-dark}",
+  "vads-segmented-control-color-surface-selected-on-dark": {
+    "name": "vads-segmented-control-color-surface-selected-on-dark",
+    "value": "{vads-color-base-dark.*.value}",
```

Comment:
Were we intentionally changing the color of the selected Segmented Control? Looks like the tokens being generated have hex #565c65 from this while the existing component has grayMedium which is hex #757575

Reply:
Thanks for double checking! Yes, this was an intentional change. Since the Segmented Control was our first component, I didn't really get to make any design improvements before we built it. Since we're updating the tokens, I decided to double check all of the contrast ratios and noticed that the grayMedium #757575 background failed with our text color in 3 of 4 tests. By darkening the selected segment to #565c65, we now pass in 3 of 4 tests.